### DISCUSSION TITLE
Related datasets

### DISCUSSION THREAD

**Discussion by miekeGR** [ternaustralia/ontology_tern#212] [Modelling]
Is there a way to capture relationships between datasets (other than void:subset) in the TERN ontology?

**Comment by smguru** [comment]
hello @miekeGR, we can add another property to showcase the relationship.

**Comment by nicholascar** [comment]
There is now basic modelling of Projects & Programs in ABIS that allows Datasets to be related to each other in terms of who created and/or managed them and for what purpose, but this is outside of the TERN Ontology so we won't be proposing that modelling is absorbed into the TERN Ont.
